airbyte-webapp/src/components/ConnectorIcon/ConnectorIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import styled from "styled-components";
+import React from "react";
+
+import { getIcon } from "utils/imageUtils";
+
+type Props = {
```

Comment:
```suggestion
interface Props {
```

I found that there's an advantage of using `interface` over `type` for Props because with vscode codelens you are able to see references to each individual field.

Reply:
Indeed, sorry that was a copy and paste error from copying that code from ImageBlock over :)